tslint.json

```diff
@@ -113,6 +113,7 @@
         "jsx-no-multiline-js": false,
         "jsx-wrap-multiline": false,
         "no-var-requires": false,
-        "no-empty": false
+        "no-empty": false,
+        "member-ordering": false
```

Comment:
That's a good discussion point, maybe we disable it right now

We have to research a bit on this and know why some people prefer to declare private then public, personally I preffer public then private as @JaimeSalas mentioned, but it's so clear to me, that I'm thinking I'm missing something, don't know why people does it the other way around.

https://palantir.github.io/tslint/rules/member-ordering/